flink-dist/pom.xml

```diff
@@ -140,6 +140,22 @@ under the License.
 			</exclusions>
 		</dependency>
 
+		<dependency>
```

Comment:
There is a section further down on the POM, with a comment "dependencies for "/examples" in scope provided. How about moving these entries there, then we get comments for "why it is like that" for free...

Reply:
sure, will move while merging!